dvc/output/base.py

```diff
@@ -282,12 +283,10 @@ def verify_metric(self):
     def download(self, to):
         self.remote.download(self.path_info, to.path_info)
 
-    def checkout(
-        self, force=False, progress_callback=None, tag=None, relink=False
-    ):
+    def checkout(self, force=False, tag=None, relink=False):
         if not self.use_cache:
-            if progress_callback:
-                progress_callback(str(self.path_info), self.get_files_number())
+            pbar = flags.tqdm or noop
```

Comment:
Maybe it would make sense to just have `pbar`  as a part of `flags` or some other structure instead of doing `flags.tqdm or noop` everywhere?